takeshape/penny
# Removing theme-ui and related deps

### Test Plan (Steps to test):

1. Click through the preview site for regressions
1. Check out the (temporary) substitute components — loyalty card, referrals, about page. These were converted just to satisfy removing theme-ui, they all should be properly updated in a future story.
